dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AbstractWorkflowResource.java

```diff
@@ -417,9 +422,19 @@ private LambdaEvent createBasicEvent(String repository, String gitReference, Str
 
                 String subclass = wf.getSubclass();
                 String workflowName = wf.getName();
+                Boolean publish = wf.getPublish();
 
                 Workflow workflow = createOrGetWorkflow(BioWorkflow.class, repository, user, workflowName, subclass, gitHubSourceCodeRepo);
                 workflow = addDockstoreYmlVersionToWorkflow(repository, gitReference, dockstoreYml, gitHubSourceCodeRepo, workflow);
+
+                if (publish != null) {
+                    try {
+                        workflow = publishWorkflow(workflow, publish);
+                    } catch (CustomWebApplicationException ex) {
+                        LOG.warn(ex.getMessage());
```

Comment:
Add a little context as in `LOG.warn("something messed up with setting publishing state", ex)`

Reply:
Addressed in latest commit